lib/Controller/SettingsController.php

```diff
@@ -81,4 +82,21 @@ public function resetAllTags(): JSONResponse {
 		$this->tagService->resetAllTags();
 		return new JSONResponse(['status' => 'success']);
 	}
+
+    public function getCounters(): JSONResponse {
+        try {
+            $filesCount = $this->tagService->getScannedFilesCount();
```

Comment:
does this actually work? because $filesCount is defined inside the try but you use it outside.

Reply:
The awesomeness of PHP makes it possible

Reply:
I tried it out. Mindblowing.

![image](https://github.com/GDATASoftwareAG/nextcloud-gdata-antivirus/assets/13208176/5f143c68-ce11-434a-9e78-53a489149e8c)